src/gui/transferlistfilterswidget.cpp

```diff
@@ -38,6 +38,7 @@
 #include <QMessageBox>
 #include <QScrollArea>
 #include <QVBoxLayout>
+#include <QUrl>
```

Comment:
Why this line?

Reply:
I removed `base/utils/misc.h` because it is not used, then found out this header is missing.